TRAMS-API/Models/Validators/PostProjectsRequestModelValidator.cs

```diff
@@ -0,0 +1,107 @@
+﻿using API.Models.Request;
+using FluentValidation;
+using System;
+using System.Linq;
+
+namespace API.Models.Validators
+{
+    public class PostProjectsRequestModelValidator : AbstractValidator<PostProjectsRequestModel>
+    {
+        public PostProjectsRequestModelValidator()
+        {
+            RuleFor(p => p.ProjectInitiatorFullName).NotEmpty().WithMessage("Must not be empty");
```

Comment:
Suggestion: Use constants for commonly used / repeated validation messages.

Reply:
Will fix